lib/perl/Genome/Qc/Tool.pm

```diff
@@ -25,7 +25,15 @@ class Genome::Qc::Tool {
 
 sub cmd_line {
     my $self = shift;
-    die $self->error_message("Abstract method run must be overriden by subclass");
+    my $cmd = $self->gmt_class->create($self->gmt_params);
+    return $cmd->build_cmdline_list;
```

Comment:
I think this method might belong in Genome::Qc::Tool::Picard - since gmt_class and build_cmdline_list aren't necessarily common interfaces for all tools, just for picard.